### ISSUE TITLE
"The easy way" missed on step

### ISSUE THREAD

**Issue by xiaobolove** [thomasstreet/famous-angular-starter#10]
the demo need Angular dependency support, to do that run "bower install angular" in the root directory to download the angular support before run "serve"

**Comment by flakyfilibuster** [comment]
hmm, there is also the assumption that bower is per default installed globally - might as well add that as an additional step beforehand

**Comment by cedric25** [comment]
I agree, I had to run a "bower install" before "serve" to make it work.
